### DISCUSSION TITLE
Welcome to maker.js Discussions!

### DISCUSSION THREAD

**Discussion by danmarshall** [microsoft/maker.js#488] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JWD-Vektor** [comment]
I want to add functionality to maker.js. To Playground in particular. I'm retired and want to give back. I am sitting on thousands of lines of Application code in the CAD/CAM arena. I have been trying for a week yto get maker.js to fully work on multiple machines and varied browsers. I have download from every link I can find. Does this even work anymore? I reaaly want to give back and this would really help me. Thank you !
